### ISSUE TITLE
Species labels being printed

### ISSUE THREAD

**Issue by mliu49** [ReactionMechanismGenerator/RMG-Py#1360]
In recent travis builds, species labels are being printed by some function. This is most notable if you look at the functional tests. I couldn't find where this was happening, so I was hoping that someone might remember leaving in a print statement...

**Comment by alongd** [comment]
Could this be it?
https://github.com/ReactionMechanismGenerator/RMG-Py/commit/a94718a2b684c4d2a70d16468dd5f78327380de3

**Comment by mliu49** [comment]
I don't think that's it. The tests normally suppress info and debug level logging statements. The output also doesn't include any other text (like 'reactants' or 'products'). Here's a snippet of the log:
```
O(2)
H2(3)
H(4)
OH(5)
HO2(6)
O2(7)
H2O2(8)
CH(9)
CO(10)
CH2(11)
HCO(12)
CH2(S)(13)
CH3(14)
CH2O(15)
CH4(16)
CO2(17)
CH2OH(18)
CH3O(19)
CH3OH(20)
C2H(21)
C2H2(22)
HCCO(23)
C2H3(24)
CH2CO(25)
C2H4(26)
C2H5(27)
H2O(28)
C(29)
HCCOH(30)
CH2CHO(31)
CH3CHO(32)
C3H8(33)
```

**Comment by mliu49** [comment]
Fixed by #1396.
